### PR TITLE
Add LLM-based fact-checker

### PR DESCRIPTION
**Pull request by Kseymur**
Bot uses WebPilot API for fact-checking (LLM+web search). Works faster, provides links to the sources used for fact-checking. Need WebPilot key

### CODE REVIEW

**Comment by Kseymur** [comment]
Update files with some improvements

**Comment by Kseymur** [comment]
Updated with refactored files + added a config file

File: tools/article_checker.py
Comment (evgenydmitriev) [inline tools/article_checker.py:153]: could you add a comment for the main that describes the functionality, detailed enough for GPT to be able to generate the working code

File: tools/config.py
Comment (evgenydmitriev) [inline tools/config.py:1]: Config files shouldn't have executable extensions. I suggest `model.config` or something similar.

**Review by evgenydmitriev — CHANGES_REQUESTED** [review]
nice! just a few cosmetic changes needed.

**Comment by Kseymur** [comment]
done

**Review by evgenydmitriev — APPROVED** [review]
great, thanks! please create a test pr to the wiki after we merge this, so we can test the factcheck command. it can be any random article, we won't merge it.

**Comment by albina-at-inca** [comment]
@Kseymur  could you email your BTC address to wiki.challenge.bounties@dn.institute ? Please include the QR code.

**Pull request merged**
